packages/sit-onyx/src/components/OnyxDataGrid/features/sorting/sorting.ts

```diff
@@ -0,0 +1,68 @@
+import { computed, h, ref } from "vue";
+import { createFeature } from "..";
+import { injectI18n } from "../../../../i18n";
+import type { DataGridEntry } from "../../types";
+import SortAction from "./SortAction.vue";
+
+export type SortDirection = "asc" | "desc" | "none";
```

Comment:
```suggestion
export type SortDirection = "asc" | "desc";
```

What do you think about making undefined the "none" state?